SoftLayer/managers/network.py

```diff
@@ -824,3 +824,15 @@ def get_closed_pods(self):
         mask = """mask[name, datacenterLongName, frontendRouterId, capabilities, datacenterId, backendRouterId,
                 backendRouterName, frontendRouterName]"""
         return self.client.call('SoftLayer_Network_Pod', 'getAllObjects', mask=mask, filter=closing_filter)
+
+    def get_datacenter(self, _filter=None, datacenter=None):
```

Comment:
I think we should remove `_filter` from the command signature, since your just setting it to None on line 833 anyway.